### ISSUE TITLE
Questions about the object detection stage 

### ISSUE THREAD

**Issue by plutozjc** [doubledaibo/drnet_cvpr2017#11]
Hello, @doubledaibo .
In the object detection stage as shown in Figure. 2, do you train the object detection model using VRD dataset, or just use the pre-trained object detection model on PASCAL VOC or MS-COCO? If you train the object detection model using VRD, could you please release it? Besides, could you please release the training code of the joint recognition stage? 

Many thanks!!

**Comment by doubledaibo** [comment]
On VRD, I used detection results provided by VRD's repo, to make a fair comparison. You could change the testing prototxt to training prototxt and train the joint recognition stage, where I found using indicator vectors is better than using the responses from the obj detector.

**Comment by plutozjc** [comment]
Thank you. @doubledaibo 
Do the indicator vectors mean groundtruth label vectors of subject and object?  When testing the relation detection, the groundtruth labels of detected subject and object are not available, should I use the predicted prob from the object detection model?

**Comment by doubledaibo** [comment]
You can use detected label to generate indicator vectors, which we found is better.
